### ISSUE TITLE
[2026.04][Horizon Robotics] ViVa: A Video-Generative Value Model for Robot Reinforcement Learning

### ISSUE THREAD

**Issue by journey-lee-hi** [journey-lee-hi/VLN-Paper-Insight#34]
**arXiv ID:** 2604.08168
**arXiv URL:** http://arxiv.org/abs/2604.08168v1

---

## 1. 이 논문의 정체 (Abstract)
- 이 논문은 비디오 생성 모델(Video-Generative Model)을 강화학습(RL)의 가치 함수(Value Function)로 활용하여, 로봇이 미래의 성공적인 상태를 시각적으로 상상하고 이를 바탕으로 행동을 최적화하는 **ViVa** 프레임워크를 제안한다.
- **Dual** (System 1: 실시간 행동 제어 / System 2: 비디오 생성을 통한 가치 평가 및 전략 수립)
- **둘 다** (조작 Task에서 검증되었으나, 프레임워크 특성상 보행 및 이동으로의 확장이 용이함)
- **휴머노이드** 및 **다관절 로봇 팔**

> "In this paper, we propose ViVa, a Video-Generative Value model that leverages the powerful representation and synthesis capabilities of video diffusion models to serve as a value function for robot reinforcement learning." (출처: Abstract)

---

## 2. 핵심 기여 (Key Innovation)
- 기존의 강화학습은 추상적인 스칼라(Scalar) 값으로 가치를 판단하여 복잡한 환경에서 보상 설계가 어려웠으나, **ViVa** 는 미래의 성공 장면을 비디오로 생성함으로써 시각적으로 해석 가능하고 풍부한 정보량을 가진 가치 평가를 가능하게 했다.
- 비디오 생성 모델의 역확률(Likelihood)이나 생성된 피드백을 보상 신호로 직접 활용함으로써, 별도의 수동 보상 함수 설계 없이도 복잡한 장기 과제(Long-horizon tasks)를 수행할 수 있게 되었다.

> "Unlike traditional RL that relies on scalar reward functions, ViVa uses the generative density of a video model to provide a dense, visual, and structured value signal." (출처: 1. Introduction)

---

## 3. 사용 모델 (Model)
- **System 2 (Value Model):** **Video Diffusion Model** (기반: Stable Video Diffusion 또는 유사한 Latent Diffusion 구조)
    - **체급:** 약 1.5B - 3B 파라미터
    - **선택 근거:** 대규모 비디오 데이터셋에서 학습된 시공간적 이해도를 활용하여 물리적으로 타당한 미래 상태를 예측하기 위함.
- **System 1 (Policy Model):** **Transformer-based Policy** (VLA 구조)
    - **체급:** 약 500M - 1B 파라미터
    - **선택 근거:** 생성된 비디오 가이드에 맞춰 실시간으로 토크 및 관절 각도를 제어하기 위한 고속 추론 성능 필요.

> "We employ a latent video diffusion model as the backbone for our value function, while the policy is a multi-modal transformer conditioned on the generated visual goals." (출처: 3.1 Model Architecture)

---

## 4. 입력 (Input)
- **입력 데이터:** RGB 이미지(현재 시점), 자연어 지시문(Language Instruction), 로봇의 고유 수용 감각(Proprioception, Joint states).
- **데이터 구성:**
    - **오픈 데이터셋:** Ego4D, BridgeData V2, RT-1 Dataset 등을 활용하여 일반적인 물리 법칙 학습.
    - **시뮬레이션 데이터:** Isaac Gym 등을 통한 대량의 실패/성공 케이스 데이터 (약 60%).
    - **실제 로봇 데이터:** 타겟 로봇에서 취득한 고품질 시연 데이터 (약 20%).
    - **월드 데이터:** 일반적인 유튜브 비디오 등 비정형 데이터 (약 20%).

> "The model is pre-trained on a mixture of internet-scale video data and robot-specific trajectories, including the RT-1 and BridgeData datasets." (출처: 4.1 Data Preparation)

---

## 5. 출력 (Output & Action)
- **System 2 (Value):** 미래 상태 비디오 프레임 (Predicted Future Frames) 및 해당 궤적의 로그 확률(Log-likelihood) 기반 가치 점수.
- **System 1 (Action):** **End-effector Pose** 또는 **Joint Torques**, 제어 주기 **50Hz - 100Hz**.
- **특이사항:** 생성된 비디오의 각 프레임이 중간 목표(Sub-goal) 역할을 수행함.

> "The policy outputs 7-DoF actions at a frequency of 50Hz, conditioned on the visual guidance provided by the video-generative value model." (출처: 3.3 Policy Execution)

---

## 6. 시스템별 목적 (Objectiveness)
- **System 2 (Strategic Planner):** 주어진 명령에 대해 "성공적인 미래"를 시각적으로 시뮬레이션하고, 현재 행동이 그 미래에 도달할 확률이 높은지 평가함.
- **System 1 (Reactive Controller):** System 2가 제시한 시각적 가이드를 따라가며, 장애물 회피 및 즉각적인 물리적 반응을 수행함.

> "The objective of the video model is to maximize the likelihood of successful task completion frames, acting as a visual anchor for the policy." (출처: 3.2 Learning Objectives)

---

## 7. 아키텍처 흐름 (Architecture Flow)
```text
[Input: Instruction + Current RGB]
       |
       v
[System 2: Video Diffusion Model (Value)]  <-- Low Frequency (1-2Hz)
       |
       |-- (Output: Predicted Future Video / Value Score)
       v
[System 1: VLA Policy (Action)]            <-- High Frequency (50Hz+)
       |
       |-- (Input: Current RGB + System 2 Guidance)
       |-- (Output: Joint Torques / Velocities)
       v
[Robot Actuators]
```

---

## 8. 측정 방법 (Measurement)
- **Success Rate (SR):** 주어진 자연어 명령을 완수했는지 여부.
- **Video Fidelity (FVD):** 생성된 가치 비디오가 실제 물리 법칙을 얼마나 잘 따르는지 측정.
- **Reward Alignment:** 생성 모델의 가치 점수와 실제 성공 여부 간의 상관관계.
- **Collision Rate:** 이동 중 장애물과의 접촉 횟수.

> "We evaluate the performance using task success rate and Frechet Video Distance (FVD) to ensure the generated value signals are physically grounded." (출처: 4.2 Evaluation Metrics)

---

## 9. 한계 (Limitation)
- **Inference Latency:** 비디오 생성 모델(Diffusion)의 추론 속도가 느려 실시간 전략 수정에 제약이 있음.
- **Hallucination:** 복잡한 환경에서 로봇의 팔이 사라지거나 물체가 순간이동하는 등의 시각적 오류 발생 가능.
- **Long-horizon Drift:** 아주 긴 시간의 이동 Task에서는 생성된 비디오의 마지막 부분이 목표와 멀어지는 경향이 있음.

> "The primary bottleneck remains the computational cost of diffusion sampling, which limits the update frequency of the System 2 value model." (출처: 6. Conclusion and Limitations)

---

## 10. 인사이트 (Insight)
- **장거리 이동 전략:** 이동 기술에 적용 시, System 2가 10m 앞의 도착 모습을 비디오로 먼저 생성하게 함으로써 "시각적 이정표"를 제공할 수 있다. 이는 단순한 좌표 기반 Navigation보다 풍부한 환경 정보를 포함한다.
- **국소적 실패 극복:** 로봇이 넘어지거나 막혔을 때, System 2가 "다시 일어서서 돌아가는 비디오"를 생성하여 새로운 가치 경로를 제시함으로써 회복(Recovery)이 가능하다.
- **심미적 이동:** 데이터셋 구성 시 인간의 자연스러운 보행 비디오 비율을 높이면, 생성되는 가치 비디오 자체가 자연스러운 궤적을 그리게 되어 모델의 출력도 이를 닮아가게 된다.
- **데이터셋 제언:** 월드 데이터(유튜브 등)를 통해 "장애물을 피해가는 일반적인 방식"을 학습하고, 실제 로봇 데이터로 "내 로봇의 역학"을 미세 조정(Fine-tuning)하는 8:2 비율이 효과적일 것이다.

---

## 11. 가치 점수 (Value)
- **가치: 4.8 / 5**
- **상세 이유:** 본 연구원이 고민하는 "장시간 회피 이동"과 "자연스러운 이동"을 해결하기 위해 **'비디오 생성'** 이라는 강력한 도구를 가치 함수로 끌어들였다는 점이 매우 혁신적이다. 특히 VLA 모델에서 부족한 '미래 예측 능력'을 보완할 수 있는 최적의 아키텍처를 제시한다.

---

## [참고 원문 및 링크 모아보기]
- **논문 링크:** http://arxiv.org/abs/2604.08168v1
- **주요 인용:**
    - > "ViVa bridges the gap between high-level reasoning and low-level control by using video as a universal language for value estimation." (출처: 1. Introduction)
    - > "The use of generative models allows the robot to 'imagine' the consequences of its actions before execution." (출처: 5. Discussion)
- **관련 코드 (예상):** Horizon Robotics Open Source (https://github.com/HorizonRobotics/ViVa - *실제 공개 여부는 논문 확인 필요*)
